services/src/main/java/uk/gov/moj/sdt/services/utils/MessagingUtility.java

```diff
@@ -81,6 +85,22 @@ public void enqueueRequest(final IIndividualRequest individualRequest) {
         queueRequest(individualRequest);
     }
 
+    @Override
+    public void enqueueRequests(List<IIndividualRequest> individualRequests) {
+        executorService.schedule(() -> queueRequest(individualRequests), 2, TimeUnit.SECONDS);
+    }
+
+    private void queueRequest(List<IIndividualRequest> individualRequests) {
+        for (IIndividualRequest iRequest : individualRequests) {
+            boolean enqueueable = iRequest.isEnqueueable();
```

Comment:
Should the boolean variable be declared before the loop rather than inside it?

Reply:
no, variable is for each request, not for the list